test/Dialect/Calyx/emit.mlir

```diff
@@ -3,7 +3,7 @@
 module attributes {calyx.metadata = ["location1", "location2"], calyx.entrypoint = "main"} {
 
   // CHECK: import "primitives/core.futil";
-  // CHECK-LABEL: component A<"static"=1>(in: 32, @go go: 1, @clk clk: 1, @reset reset: 1) -> (out: 32, @done done: 1) {
+  // CHECK-LABEL: component A<"static"=1,>(in: 32, @go go: 1, @clk clk: 1, @reset reset: 1) -> (out: 32, @done done: 1) {
```

Comment:
Are these commas after attributes intentional?

Reply:
Yup, unfortunately couldn't figure out a nice way to remove these. There is a `pop_back` call but I think its getting iffy to do this printing. Ideally we'd want to use `llvm::interleave` but we're also doing filtering while printing and I couldn't figure out a nice way to write that code here.